src/sdc11073/xml_types/xml_structure.py

```diff
@@ -1381,25 +1469,20 @@ class NodeTextQNameListProperty(_ElementListProperty):
     The xml text is the joined list of qnames in the form prefix:localname.
     """
 
-    def __init__(self, sub_element_name: etree.QName | None,
-                 is_optional: bool = False):
-        super().__init__(sub_element_name, ListConverter(ClassCheckConverter(etree.QName)),
-                         is_optional=is_optional)
+    def __init__(self, sub_element_name: etree.QName | None, is_optional: bool = False):
+        super().__init__(sub_element_name, ListConverter(ClassCheckConverter(etree.QName)), is_optional=is_optional)
 
     def get_py_value_from_node(self, instance: Any, node: xml_utils.LxmlElement) -> Any:  # noqa: ARG002
```

Comment:
typing of return value should be something like:

list(xml_utils.QName) | None

Reply:
done